nomenclature/region_mapping_models.py

```diff
@@ -151,6 +153,20 @@ def create_from_region_mapping(cls, file: Union[Path, str]):
             mapping_input["common_regions"] = common_region_list
         return cls(**mapping_input)
 
+    @property
+    def region_names_all(self) -> List[str]:
+        nr_list = (
+            [x.target_native_region for x in self.native_regions]
+            if self.native_regions is not None
+            else []
```

Comment:
Shouldn't this work?
```suggestion
            [x.target_native_region for x in self.native_regions or list()]
```

Reply:
Ah nice, that should work.